migrations/versions/18d1e622ccc5_add_wheatley_columns_to_database.py

```diff
@@ -0,0 +1,30 @@
+"""Add Wheatley columns to database
+
+Revision ID: 18d1e622ccc5
+Revises: fd24fcbb6ba6
+Create Date: 2020-09-03 22:21:15.965629
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '18d1e622ccc5'
+down_revision = 'fd24fcbb6ba6'
+branch_labels = None
+depends_on = None
```

Comment:
I get `flask db upgrade`
> ERROR [root] Error: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

 Does this need to be regenerated from the latest code?
```
flask db history
fd24fcbb6ba6 -> 18d1e622ccc5 (head), Add Wheatley columns to database
fd24fcbb6ba6 -> e1634351a997 (head), user tokens
22cc99c77bd3 -> fd24fcbb6ba6 (branchpoint), host mode enabled
ecf0969f42be -> 22cc99c77bd3, track tower created date, user joined date
```

Reply:
I recommend running `flask db merge heads`, which will effectively merge the two branches.

Reply:
Yeah I also found that error and I think I got round it by specifying the exact commit I wanted to use.